gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java

```diff
@@ -288,6 +304,32 @@ private String determineEndpoint() throws IOException {
       return endpoint;
     }
 
+    /** Determine if S2A can be used */
+    @VisibleForTesting
+    boolean shouldUseS2A() {
+      // If EXPERIMENTAL_GOOGLE_API_USE_S2A is not set to true, skip S2A.
+      String s2AEnv;
```

Comment:
Can we add a comment that this environment variable is temporary and we'll remove it in the future?

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3326/commits/6eeccb27cce567a3856cde06ad8ba055b754daa4

Also updated description similar to description to indicate when S2A will be non-experimental